app/models/repository_rows_my_module.rb

```diff
@@ -0,0 +1,7 @@
+class RepositoryRowsMyModule < ActiveRecord::Base
```

Comment:
I think Rails has special rules for naming join models, which represents many to many association and I think both sides of association should be plural